core/runtime.rs

```diff
@@ -841,7 +841,14 @@ impl JsRuntime {
           )
           .await?;
         let receiver = runtime.mod_evaluate(id);
-        runtime.run_event_loop(false).await?;
+        poll_fn(|cx| {
+          let r = runtime.poll_event_loop(cx, false);
+          // FIXME(bartlomieju): some code in readable-stream polyfill in `ext/node`
+          // is calling `nextTick()` during snapshotting, which causes infinite loop
+          runtime.state.borrow_mut().has_tick_scheduled = false;
+          r
```

Comment:
@kt3k is this something you could look into? It's coming from the minified code in `ext/node/polyfills/_stream.mjs` but I wasn't able to find exactly where. We should change the code to not call `nextTick` on first execution (probably some lazy initialization is needed).

Reply:
It's not a blocker to land this PR, we can fix it separately.